### PR TITLE
Maintainers += Richard Si (aka ichard26)

### PR DESCRIPTION
**Pull request by ichard26**
pretty self-explanatory (if not, go look at the branch name)

### CODE REVIEW

**Review by JelleZijlstra — APPROVED** [review]

**Comment by JelleZijlstra** [comment]
Now go push that fancy new button :)

File: README.md [hunk 1]
```diff
@@ -539,8 +539,8 @@ Maintained with [Carol Willing](mailto:carolcode@willingconsulting.com),
 [Carl Meyer](mailto:carl@oddbird.net),
 [Jelle Zijlstra](mailto:jelle.zijlstra@gmail.com),
 [Mika Naylor](mailto:mail@autophagy.io),
-[Zsolt Dollenstein](mailto:zsol.zsol@gmail.com), and
-[Cooper Lees](mailto:me@cooperlees.com).
+[Zsolt Dollenstein](mailto:zsol.zsol@gmail.com),
+[Cooper Lees](mailto:me@cooperlees.com), and Richard Si.
```
Comment (cooperlees) [inline README.md:543]: You don't want your own line?

**Review by cooperlees — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (ichard26) [reply README.md:543]: prettier hates that

**Comment by ichard26** [comment]
because i'm paranoid, I'mma wait for CI to go green, good habit to start building anyway

**Comment by ichard26** [comment]
Okay, I've done enough waiting, let's press this shiny green button :)

**Comment by ichard26** [comment]
that felt *good* :tada:

**Pull request merged**